tin/static/css/edit.css

```diff
@@ -0,0 +1,87 @@
+.collapsible {
+  background-color: #777;
+  color: white;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
+  border: none;
+  text-align: center;
+  outline: none;
+  font-size: 15px;
+}
+
+.active, .collapsible:hover {
+  background-color: #555;
+}
```

Comment:
I'm not sure what I think of the styling... it does seem out of place with the rest of Tin, but looks very cool. If you think you can get it to fit in a bit more, go ahead, but it's probably good enough already (almost anything is better than the current).

Reply:
I tried adding some neomorphic effects, not a very artistic person :)